website/docs/d/federated_query_limits.html.markdown

```diff
@@ -18,7 +18,7 @@ description: |-
 ```terraform
 data "mongodbatlas_federated_query_limits" "test" {
   project_id = "PROJECT_ID"
-  tenant_name = "FEDERATED_DATABSE_INSTANCE_NAME"
+  tenant_name = "FEDERATED_DATABASE_INSTANCE_NAME"
```

Comment:
Not blocking but nit - we have one style in one data source in another in this one.   We really need to settle on this with a style guide.   I personally prefer real values but I'm okay with this format if the team/docs think it's better.   But either way we should  find a way to be consistent. cc @Zuhairahmed

Reply:
Sounds good and agree @themantissa. Found time in a few weeks with Docs Team and others to align on both a style guide and after that go through each resource by resource to make sure we are being consistent.